Userland/Libraries/LibJS/Heap/HeapBlock.h

```diff
@@ -96,16 +96,14 @@ class HeapBlock {
         return reinterpret_cast<Cell*>(&m_storage[index * cell_size()]);
     }
 
-    FreelistEntry* init_freelist_entry(size_t index)
-    {
-        return new (&m_storage[index * cell_size()]) FreelistEntry();
-    }
```

Comment:
This could just as well be a separate commit :^)

Reply:
Good point, drive-by is not a good thing. I split it out.